paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -197,7 +197,7 @@ internal class PaymentOptionsViewModel @Inject constructor(
     }
 
     override fun onPaymentResult(paymentResult: PaymentResult) {
-        _processing.value = false
+        savedStateHandle[SAVE_PROCESSING] = false
```

Comment:
Really just a temporary change until `PaymentSheetState` is backing these properties.